utils/doclint/missingDocs.js

```diff
@@ -22,7 +22,7 @@ const path = require('path');
 
 /** @typedef {import('../../markdown').MarkdownNode} MarkdownNode */
 
-const IGNORE_CLASSES = ['PlaywrightAssertions', 'GenericAssertions', 'LocatorAssertions', 'PageAssertions', 'APIResponseAssertions', 'SnapshotAssertions'];
+const IGNORE_CLASSES = ['PlaywrightAssertions', 'GenericAssertions', 'LocatorAssertions', 'PageAssertions', 'APIResponseAssertions', 'SnapshotAssertions', 'PageError'];
```

Comment:
@dgozman Any idea how to solve this erorr from docs lint? 
```
> node utils/doclint/cli.js

ERROR: missing documentation:
Documented "PageError" not found in sources
```

Reply:
You can add `export type { PageError } from './pageError';` into `packages/playwright-core/src/client/api.ts` and then the error is gone.